src/views/Expenses.js

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import { v4 as uuid } from 'uuid';
+import AddExpensesForm from '../components/AddExpensesForm';
+import ExpensesList from '../components/ExpensesList';
+
+const Expenses = () => {
+  const [expenses, setExpenses] = useState([]);
+
+  // Run on initial render only
+  useEffect(() => {
+    const token = process.env.REACT_APP_MOSTASH_API_KEY;
+    const url = `http://pi.motine.de:12305/items.json?stash=${token}&kind=expense`;
+    fetch(url)
+      .then(response => response.json())
+      .then(data => setExpenses(data))
+      .catch(error => console.log('error', error));
+  }, []);
```

Comment:
This works. Fetches expenses from mostash on initial render and pushes them to state. To see the add/delete functionality of the app in action (without any mostash data persistence) comment out lines 10-17.